src/main/java/edu/kis/vh/nursery/collections/IntArrayStack.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery.collections;
+
+public class IntArrayStack implements Stack{
+
+    private static final int STACK_CAPACITY = 12;
+    private static final int FULL_STACK = 11;
```

Comment:
PLEASE FIX: This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).